yt/docker/ya-build/query-tracker/build_query_tracker.sh

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
+
+# This script is expected to be executed from `build_query_tracker_in_docker.sh` script. See it for more details.
+
+set -eux
+shopt -s expand_aliases
+
+export YTSAURUS_SOURCE_PATH=$(realpath $YTSAURUS_SOURCE_PATH)
+export YDB_SOURCE_PATH=$(realpath $YDB_SOURCE_PATH)
+export YTSAURUS_BUILD_PATH=$(realpath $YTSAURUS_BUILD_PATH)
+export YQL_BUILD_PATH=$(realpath $YQL_BUILD_PATH)
+export OUTPUT_PATH=$(realpath $OUTPUT_PATH)
+
+mkdir -p $YTSAURUS_BUILD_PATH
+mkdir -p $YQL_BUILD_PATH
+mkdir -p $OUTPUT_PATH
+
+apt-get update
+DEBIAN_FRONTEND=noninteractive TZ=Etc/UTC apt-get install -y software-properties-common build-essential
+DEBIAN_FRONTEND=noninteractive TZ=Etc/UTC add-apt-repository -y ppa:deadsnakes/ppa
+apt-get update
+DEBIAN_FRONTEND=noninteractive TZ=Etc/UTC apt-get install -y python3.11-dev
+
+# Copy required files from ytsaurus source
+mkdir -p ${OUTPUT_PATH}/yt/python
+cp -a ${YTSAURUS_SOURCE_PATH}/yt/python/. ${OUTPUT_PATH}/yt/python
+mkdir -p ${OUTPUT_PATH}/yt/yt/python
+cp -a ${YTSAURUS_SOURCE_PATH}/yt/yt/python/. ${OUTPUT_PATH}/yt/yt/python
+mkdir -p ${OUTPUT_PATH}/certs
+cp ${YTSAURUS_SOURCE_PATH}/certs/cacert.pem ${OUTPUT_PATH}/certs/cacert.pem
+
+# Build python libraries
+python3 ${YTSAURUS_SOURCE_PATH}/ya make \
+    ${YTSAURUS_SOURCE_PATH}/yt/yt/python/yson_shared \
+    ${YTSAURUS_SOURCE_PATH}/yt/yt/python/driver/rpc_shared \
+    ${YTSAURUS_SOURCE_PATH}/yt/yt/python/driver/native_shared \
+    -T --ignore-recurses ${BUILD_FLAGS} -DSTRIP=yes \
+    -DOS_SDK=local -DUSE_ARCADIA_PYTHON=no -DUSE_LOCAL_PYTHON -DPYTHON_CONFIG=python3.11-config -DPYTHON_BIN=python3.11 \
+    --output ${YTSAURUS_BUILD_PATH}
+
+strip --remove-section=.gnu_debuglink ${YTSAURUS_BUILD_PATH}/yt/yt/python/yson_shared/*.so ${YTSAURUS_BUILD_PATH}/yt/yt/python/driver/rpc_shared/*.so ${YTSAURUS_BUILD_PATH}/yt/yt/python/driver/native_shared/*.so
+
+mkdir -p ${OUTPUT_PATH}/artifacts
+
+cp ${YTSAURUS_BUILD_PATH}/yt/yt/python/yson_shared/yson_lib.so ${OUTPUT_PATH}/artifacts/libyson_lib.so
+cp ${YTSAURUS_BUILD_PATH}/yt/yt/python/driver/native_shared/driver_lib.so ${OUTPUT_PATH}/artifacts/libdriver_lib.so
+cp ${YTSAURUS_BUILD_PATH}/yt/yt/python/driver/rpc_shared/driver_rpc_lib.so ${OUTPUT_PATH}/artifacts/libdriver_rpc_lib.so
+
+# Build ytserver-all
+${YTSAURUS_SOURCE_PATH}/ya make -T ${BUILD_FLAGS} --output=${YTSAURUS_BUILD_PATH} ${YTSAURUS_SOURCE_PATH}/yt/yt/server/all
```

Comment:
Does this mean that from this moment we will start building yt/yt/server/all twice?

Reply:
No. This script is not used in CI. CI uses `build_yql_agent.sh` script and does `build_query_tracker.sh` parts separately (building python packages and ytserver-all)